staging/cse/windows/kubeletfunc.ps1

```diff
@@ -182,7 +182,18 @@ function Get-KubePackage {
         [Parameter(Mandatory = $true)][string]
         $KubeBinariesSASURL
     )
-    Logs-To-Event -TaskName "AKS.WindowsCSE.DownloadKubletBinaries" -TaskMessage "Start to download kubelet binaries and unzip. KubeBinariesPackageSASURL: $global:KubeBinariesPackageSASURL"
+    $mappingFile = [Io.path]::Combine($global:CacheDir, "private-packages\mapping.json")
+    if (Test-Path $mappingFile) {
+        $urls = @{}
+        (ConvertFrom-Json ((Get-Content $mappingFile -ErrorAction Stop) | Out-String)).psobject.properties | Foreach { $urls[$_.Name] = $_.Value }
+        if ($urls.ContainsKey($global:KubeBinariesVersion)) {
+            Write-Log "Found $global:KubeBinariesVersion in $mappingFile"
+            $KubeBinariesSASURL = $urls[$global:KubeBinariesVersion]

```

Comment:
Looks like this script will be executed per node, do we have different urls for different clouds, like mooncake, fairfax and airgap?

Reply:
We do not need any urls for different clouds since these packages are cached so it will not be downloaded